aws/logs_monitoring/lambda_function.py

```diff
@@ -465,6 +486,34 @@ def split(events):
     return metrics, logs
 
 
+# should only be called when INCLUDE_AT_MATCH and/or EXCLUDE_AT_MATCH exist
+def filter_logs(logs):
+    """
+    Applies log filtering rules.
+    If no filtering rules exist, return all the logs.
+    """
+    if INCLUDE_AT_MATCH is None and EXCLUDE_AT_MATCH is None:
+        return logs 
+    # Add logs that should be sent to logs_to_send
+    logs_to_send = []
+    # Test each log for exclusion and inclusion, if the criteria exist
+    for log in logs:
+        try:
+            string_log = json.dumps(log)
```

Comment:
Here we are transforming logs to json-string and storing the logs at the end, could we store a json-string instead and change the logic downstream to manipulation string instead of objects so that we don't do the object-to-string operation twice ?

Reply:
I went for it and made a few downstream changes.  Is the change I made in `DatadogHTTPClient` ok:  `data=self._scrubber.scrub("[{}]".format(", ".join(map(json.dumps, logs))))`?